VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/host-hearing-list/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { ProfileService } from 'src/app/services/api/profile.service';
+import { VideoWebService } from 'src/app/services/api/video-web.service';
+import { ConferenceForHostResponse, LoggedParticipantResponse, UserProfileResponse } from 'src/app/services/clients/api-client';
+import { ErrorService } from 'src/app/services/error.service';
+import { EventsService } from 'src/app/services/events.service';
+import { HearingVenueFlagsService } from 'src/app/services/hearing-venue-flags.service';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { vhContactDetails } from 'src/app/shared/contact-information';
+import { ScreenHelper } from 'src/app/shared/screen-helper';
+import { HostHearingListComponent } from 'src/app/on-the-day/host-hearing-list/host-hearing-list.component-base';
+
+@Component({
+    selector: 'app-judge-hearing-list',
+    templateUrl: '../host-hearing-list.component.html',
+    styleUrls: ['../host-hearing-list.component.scss']
+})
+export class JudgeHearingListComponent extends HostHearingListComponent implements OnInit, OnDestroy {
+    contact = {
+        phone: vhContactDetails.phone
+    };
```

Comment:
This appears to be in the base class, could this be removed?

Reply:
I will check and see what happens when I remove it, but should work with it having removed

Reply:
works just fine, removing it with my next commit